wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/scm/PersonalAccessTokenManager.java

```diff
@@ -51,4 +51,18 @@ PersonalAccessToken fetchAndSave(Subject cheUser, String scmServerUrl)
   Optional<PersonalAccessToken> get(Subject cheUser, String scmServerUrl)
       throws ScmConfigurationPersistenceException, ScmUnauthorizedException,
           ScmCommunicationException;
+
+  /**
+   * Gets {@link PersonalAccessToken} from permanent storage.
+   *
+   * @param cheUser
+   * @param scmServerUrl
+   * @param validate
```

Comment:
What does `validate` mean?
Pls, provide meaningful descriptions for params

Reply:
done